test/query_test.exs

```diff
@@ -694,4 +694,25 @@ defmodule QueryTest do
     """, [])
     assert [[1], [2]] = query("SELECT raise_notice_and_return(x) FROM generate_series(1, 2) AS x", [])
   end
+
+  test "executes query in chunks", context do
+    assert transaction(fn(conn) ->
+      assert {:ok, q} = P.prepare( conn, "", "SELECT * FROM generate_series( 1, 3 )", max_rows: 1 )
+      assert {:ok, %Postgrex.Result{rows: [[ 1 ]], query: q} } = P.execute( conn, q, [] )
+      assert {:ok, %Postgrex.Result{rows: [[ 2 ]], query: q} } = P.execute( conn, q, [] )
```

Comment:
then it _does not work_ ?

see [can nest chunked queries when they use different portals](https://github.com/thenrio/postgrex/commit/d88ef0edb33210869b2973ae19374b892b057a6e#diff-c7c0f9f6218a73a7c8312d9305570b63R719) ?

using named portal is an option to have it green